l10n_ch_import_cresus/tests/test_import.py

```diff
@@ -42,32 +47,64 @@ def setUp(self):
     def test_import(self):
         journal_obj = self.env['account.journal']
         misc = journal_obj.search(
-            [('name', 'ilike', 'miscellaneous')], limit=1)
+            [('code', '=', 'MISC')], limit=1)
+        if not misc:
+            misc = journal_obj.create({
+                'code': 'MISC',
+                'name': 'dummy MISC',
+                'type': 'general'})
+
+        def get_path(filename):
+            res = get_resource_path('l10n_ch_import_cresus', 'tests', filename)
+            return res
+
+        with open(get_path('input.csv')) as src:
+            buf = StringIO()
+            base64.encode(src, buf)
+            contents = buf.getvalue()
+            buf.close()
 
-        test_file_path = get_module_resource('l10n_ch_import_cresus',
-                                             'tests',
-                                             'cresus.txt')
-        buf = StringIO()
-        with open(test_file_path) as f:
-            base64.encode(f, buf)
-        contents = buf.getvalue()
-        buf.close()
         wizard = self.env['account.cresus.import'].create({
             'journal_id': misc.id,
             'file': contents})
         wizard._import_file()
 
         res = wizard.imported_move_ids
-        self.assertEqual(len(res), 5)
         res.assert_balanced()
-        for l in res[0].line_ids:
-            self.assertEqual(l.analytic_account_id, self.reserve)
-        self.assertEqual(res[1].date, '2002-01-07')
-        self.assertEqual(res[2].date, '2002-01-07')
-        self.assertEqual(res[3].line_ids[0].account_id.code, '6642')
-        self.assertEqual(res[3].line_ids[0].debit, 15.0)
-        self.assertEqual(res[3].line_ids[0].credit, 0.0)
-        self.assertEqual(res[3].line_ids[1].account_id.code, '1010')
-        self.assertEqual(res[3].line_ids[1].debit, 0.0)
-        self.assertEqual(res[3].line_ids[1].credit, 15.0)
-        self.assertEqual(res[4].line_ids[1].tax_line_id, self.vat)
+
+        gold = open(get_path('golden-output.txt'))
```

Comment:
This file is opened but not closed, you might want to use `with` syntax or call `close()`